doc/api/deprecations.md

```diff
@@ -3184,8 +3184,9 @@ changes:
 
 Type: Documentation-only
 
-`code` values other than `undefined`, `null`, integer numbers and integer
-strings (e.g., '1') are deprecated as parameter in [`process.exit()`][].
+Values other than `undefined`, `null`, integer numbers and integer strings
+(e.g., '1') are deprecated as the `code` parameter in [`process.exit()`][] or a
+setting of [`process.exitCode`][].
```

Comment:
I don't find "a setting of `process.exitCode`" quite clear, the AST calls it an `AssignmentExpression`, maybe we can reuse this wording?
```suggestion
(e.g., `'1'`) are deprecated as value for the `code` parameter in
[`process.exit()`][] and as value to assign to [`process.exitCode`][].
```

Reply:
Applied the review, PTAL.